src/core/public/chrome/nav_links/nav_link.ts

```diff
@@ -44,6 +44,12 @@ export interface ChromeNavLink {
    */
   readonly baseUrl: string;
 
+  /**
+   * The route used to open the {@link AppBase.defaultPath | default path } of an application.
+   * If unset, `baseUrl` will be used instead.
+   */
+  readonly url?: string;
```

Comment:
not `@deprecated` anymore?

Reply:
Correct. The deprecation on this property was added at a time we thought the `subUrlPath` was a feature that was going to be unused/removed in KP, and this property was only used by legacy apps.  It's no longer the case, and we now need distinct `baseUrl` and `url` props on navlinks even for KP apps.